docs/UserGuide.md

```diff
@@ -652,23 +661,29 @@ Other Examples:
 You can use this command to edit the attendance of a student.
 
 Format: `edit-attendance c\CLASS_INDEX l\LESSON_INDEX s\STUDENT_INDEX w\WEEK p\PARTICIPATION_SCORE`
+* Edits the attendance record at the specified `CLASS_INDEX` + `LESSON_INDEX` + `STUDENT_INDEX` + `WEEK`.
 
 For example:
 * Alex is a student in your CS2103T class. After your lesson in week 1, you wish to give him a participation score of 45.
-However, you made a typo error and gave him 35 instead. Hence, you can type in the command `edit-attendance c\1 l\1 s\1 w\1 p\45`
-and press <kbd>Enter</kbd>.
+  However, you made a typo error and gave him 35 instead. 
```

Comment:
I think typo already suggests that its an error? 

```suggestion
  However, you made a typo and gave him 35 instead. 
```
Alternatively maybe we can drop the typo word?

```suggestion
  However, you accidentally gave him a score of 35 instead.
```